### PR TITLE
HBASE-25552 Upgrade jetty jar to fix CVE-2020-27218

### PR DESCRIPTION
**Pull request by pankaj72981**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 42s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 53s |  master passed  |
| +1 :green_heart: |  compile  |   0m 12s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m  7s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 30s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 10s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 10s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  0s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m  6s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   3m 49s |  root in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m  9s |  The patch does not generate ASF License warnings.  |
|  |   |   7m 48s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hbase.htrace_noop.TestHBaseTestingUtilityWithHTraceNoop |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/1/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-thirdparty/pull/46 |
| Optional Tests | dupname asflicense javac javadoc unit xml compile |
| uname | Linux c90905ee9a21 5.4.0-1025-aws #25~18.04.1-Ubuntu SMP Fri Sep 11 12:03:04 UTC 2020 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | master / a8e3629 |
| Default Java | Oracle Corporation-1.8.0_282-b08 |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/1/artifact/yetus-precommit-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/1/testReport/ |
| Max. process+thread count | 399 (vs. ulimit of 1000) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/1/console |
| versions | git=2.20.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by busbey** [comment]
Test failure is unrelated

File: pom.xml [hunk 1]
```diff
@@ -136,7 +136,7 @@
     <commons-collections.version>4.4</commons-collections.version>
     <error_prone_annotations.version>2.3.4</error_prone_annotations.version>
     <gson.version>2.8.5</gson.version>
-    <jetty.version>9.4.34.v20201102</jetty.version>
+    <jetty.version>9.4.35.v20201120</jetty.version>
```
Comment (busbey) [inline pom.xml:139]: Can we go directly to 9.4.36.v20210114?

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 34s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| -1 :x: |  mvninstall  |   0m  2s |  root in master failed.  |
| -1 :x: |  compile  |   0m 18s |  root in master failed.  |
| -1 :x: |  javadoc  |   0m 17s |  root in master failed.  |
||| _ Patch Compile Tests _ |
| -1 :x: |  mvninstall  |   0m  6s |  root in the patch failed.  |
| -1 :x: |  compile  |   0m 18s |  root in the patch failed.  |
| -1 :x: |  javac  |   0m 18s |  root in the patch failed.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| -1 :x: |  javadoc  |   0m 17s |  root in the patch failed.  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   0m  9s |  root in the patch failed.  |
| +0 :ok: |  asflicense  |   0m 17s |  ASF License check generated no output?  |
|  |   |   3m 42s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-thirdparty/pull/46 |
| Optional Tests | dupname asflicense javac javadoc unit xml compile |
| uname | Linux f1721d58d915 5.4.0-1025-aws #25~18.04.1-Ubuntu SMP Fri Sep 11 12:03:04 UTC 2020 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | master / a8e3629 |
| Default Java | Oracle Corporation-1.8.0_282-b08 |
| mvninstall | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/artifact/yetus-precommit-check/output/branch-mvninstall-root.txt |
| compile | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/artifact/yetus-precommit-check/output/branch-compile-root.txt |
| javadoc | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/artifact/yetus-precommit-check/output/branch-javadoc-root.txt |
| mvninstall | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/artifact/yetus-precommit-check/output/patch-mvninstall-root.txt |
| compile | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/artifact/yetus-precommit-check/output/patch-compile-root.txt |
| javac | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/artifact/yetus-precommit-check/output/patch-compile-root.txt |
| javadoc | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/artifact/yetus-precommit-check/output/patch-javadoc-root.txt |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/artifact/yetus-precommit-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/testReport/ |
| Max. process+thread count | 28 (vs. ulimit of 1000) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/2/console |
| versions | git=2.20.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  3s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 27s |  master passed  |
| +1 :green_heart: |  compile  |   0m 12s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m  6s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 27s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 12s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 12s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m  5s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 33s |  root in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m  5s |  The patch does not generate ASF License warnings.  |
|  |   |   3m 22s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/3/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-thirdparty/pull/46 |
| Optional Tests | dupname asflicense javac javadoc unit xml compile |
| uname | Linux 75383f9adec0 5.4.0-1025-aws #25~18.04.1-Ubuntu SMP Fri Sep 11 12:03:04 UTC 2020 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | master / a8e3629 |
| Default Java | Oracle Corporation-1.8.0_282-b08 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/3/testReport/ |
| Max. process+thread count | 390 (vs. ulimit of 1000) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/3/console |
| versions | git=2.20.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: pom.xml (hunk 1 above)
Comment (jojochuang) [reply pom.xml:139]: +1 we should use the latest version

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 40s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 40s |  master passed  |
| +1 :green_heart: |  compile  |   0m 11s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m  7s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 24s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 11s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 11s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m  5s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 33s |  root in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m  7s |  The patch does not generate ASF License warnings.  |
|  |   |   3m  6s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/4/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-thirdparty/pull/46 |
| Optional Tests | dupname asflicense javac javadoc unit xml compile |
| uname | Linux daca0dab54a7 5.4.0-1025-aws #25~18.04.1-Ubuntu SMP Fri Sep 11 12:03:04 UTC 2020 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | master / a8e3629 |
| Default Java | Oracle Corporation-1.8.0_282-b08 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/4/testReport/ |
| Max. process+thread count | 413 (vs. ulimit of 1000) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-Thirdparty-PreCommit/job/PR-46/4/console |
| versions | git=2.20.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by jojochuang — APPROVED** [review]

**Review by busbey — APPROVED** [review]

**Pull request closed without merging**